src/Symfony/Component/Translation/Dumper/XliffFileDumper.php

```diff
@@ -40,6 +50,21 @@ public function dump(MessageCatalogue $messages, $options = array())
     }
 
     /**
+     * Set XLIFF tool element metadata.
+     *
+     * @link http://docs.oasis-open.org/xliff/v1.2/os/xliff-core.html#tool_elem
+     *
+     * @param array attributes supported by the <tool> element
+     */
+    protected function setToolMetadata( array $meta ){
```

Comment:
As per my [initial comment](https://github.com/symfony/symfony/issues/15190) I thought the maintainers would prefer to decide on best practice. I'm not experienced in contributing to Symfony, as evidenced by this discussion. If anyone maintaining the Translation component is interested in implementing this properly then great. Either way, feel free to close my PR. I'm afraid I don't have the time to rework it.